npm/react/package.json

```diff
@@ -6,7 +6,10 @@
   "scripts": {
     "build": "rimraf dist && yarn transpile",
     "build-prod": "yarn build",
-    "cy:open": "node ../../scripts/cypress open",
+    "cy:open": "node ../../scripts/start.js --component-testing --project ${PWD}",
+    "cy:open:debug": "NODE_OPTIONS=--max-http-header-size=1048576 node --inspect-brk ../../scripts/start.js --component-testing --project ${PWD}",
+    "cy:run": "node ../../scripts/start.js --component-testing --run-project ${PWD}",
+    "cy:run:debug": "NODE_OPTIONS=--max-http-header-size=1048576 node --inspect-brk ../../scripts/start.js --component-testing --run-project ${PWD}",
```

Comment:
Should we use vscode terminal tasks instead of npm scripts for this?

Reply:
Should move these to `packages/server-ct` and have something like `cy:open:react`, `cy:open:vue` etc.

Reply:
Yeah I think we should hook up all of these in server-ct shortly.